watcher/local/watcher.go

```diff
@@ -217,6 +217,12 @@ func (w *Watcher) handleEventsFromChain(eventsFromChainSub channel.AdjudicatorSu
 					log.Debug("Registered successfully")
 				}
 			}()
+		case *channel.ProgressedEvent:
```

Comment:
I was just wondering: Why is the watcher forwarding events to the client at all? Isn't a watcher usually just a consumer?

Reply:
Events are forwarded to the client as an information. Because client could act based on this information.

1. When a state is registered, it can stop creating newer off-chain states.
2. Optionally, it could also progress or settle the channel.